packages/connect-explorer/src/pages/methods/device/loadDevice.mdx

```diff
@@ -0,0 +1,82 @@
+import { Callout } from 'nextra/components';
+
+import { PROTO } from '@trezor/connect/src/constants';
+
+import { ParamsTable } from '../../../components/ParamsTable';
+import { CommonParamsLink } from '../../../components/CommonParamsLink';
+import { ApiPlayground } from '../../../components/ApiPlayground';
+import loadDevice from '../../../data/methods/management/loadDevice.ts';
+
+<ApiPlayground options={[{ title: 'Load device', legacyConfig: loadDevice[0] }]} />
+
+export const paramDescriptions = {
+    mnemonics: '',
+    label: '',
+    u2fCounter: 'Default value is set to current time stamp in seconds.',
+    pin: '',
+    passphraseProtection: '',
+    skipBackup: '',
+    noBackup: 'create a seedless device',
+};
+
+## Load device
+
+<Callout type="error">
+    **Management command** - this method is restricted to Trezor.io and can't be used in 3rd party
+    applications.
+</Callout>
```

Comment:
maybe also a note that it must be a special debug build. so that people are not confused that is possible to input seed into trezor from outside

Reply:
Sure, I added it there.